studiopress/simple-social-icons
# Add escaping where needed

- [x] Fix `composer phpcs` issues
- [x] Add more escaping where needed 
- [x] Bump version to `3.1.0` in preparation for a release

### How to test

1. No build steps needed for this plugin, like `npm run dev`, etc…
2. Activate Twenty Twenty-One, or any non-FSE theme
3. Go to the Customizer
4. Add the 'Simple Social Icons' widget
5. Populate all of the fields. For testing, URLs can all simply be `https://example.com`.
6. Expected: The widget's markup has the edits that you made in the fields: 
<img width="778" alt="Screen Shot 2022-04-04 at 4 44 17 PM" src="https://user-images.githubusercontent.com/4063887/161637439-4b9b05e2-01a1-4a9e-b066-0999767662a8.png">

### Documentation
 No documentation required (I think)



### Suggested changelog entry

- Add escaping to the widget's output
